qemu/tests/cfg/block_discard_hotplug.cfg

```diff
@@ -0,0 +1,37 @@
+- block_discard_hotplug:
+    no Windows
+    type = block_discard_hotplug
+    data_tag = "stg1"
+    images += " stg0 ${data_tag}"
+    force_create_image_stg0 = yes
+    image_size_stg0 = 1G
+    image_name_stg0 = images/stg0
+    drv_extra_params_stg0 += "discard=unmap"
+    boot_drive_stg1 = no
+    disk_serial = TARGET_DISK1
```

Comment:
Looks like we don't need `disk_serial = TARGET_DISK1` any more.

Reply:
updated